### PR TITLE
[ Bento-1800 ] PR for Login & Registration Pages. 

### PR DESCRIPTION
**Pull request by iksheth**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested?

### CODE REVIEW

File: src/utils/graphqlClient.js [hunk 1]
```diff
@@ -1,10 +1,28 @@
...
+});
+
+const authService = new HttpLink({
+  uri: AUTH_FORCE_DEV ? AUTH_DEV : AUTH,
+});
 
 const client = new ApolloClient({
   cache: new InMemoryCache(),
-  uri: BACKEND,
+  link: ApolloLink.split(
```
Comment (iksheth) [inline src/utils/graphqlClient.js:21]: @AjayDoddapaneni This is the change using which we can fire queries to different API endpoints for GraphQL.

File: src/pages/accessManagment/userRegistration/view.js [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+  // Init state for inputs.
+  const [formValues, setFormValues] = useState(setDefaultValues());
+
+  const clearAll = () => {
+    setFormValues(setDefaultValues());
+  };
+
+  // GraphQL Operations
+  const [mutate, response] = useMutation(SUBMIT_REGISTER, {
+    context: { clientName: 'authService' },
```
Comment (iksheth) [inline src/pages/accessManagment/userRegistration/view.js:50]: @AjayDoddapaneni We are passing name of client in options in useMutation hook.

File: src/bento/userLoginData.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+import LoginGov from '../assets/login/login.gov.png';
+import NihItrust from '../assets/login/nih_itrust.png';
+import { registrationRoute } from './siteWideConfig';
+
+export const pageTitle = 'Login or Register';
+
+export const loginProvidersData = {
+  google: {
+    key: 'google',
+    icon: Google,
```
Comment (AjayDoddapaneni) [inline src/bento/userLoginData.js:11]: can we pull in logo icons from datacommons assets

File: src/utils/graphqlClient.js [hunk 4]
```diff
@@ -1,10 +1,28 @@
-import { ApolloClient, InMemoryCache } from '@apollo/client';
+import {
+  ApolloClient, InMemoryCache, ApolloLink, HttpLink,
+} from '@apollo/client';
 import env from './env';
 
 const BACKEND = env.REACT_APP_BACKEND_API;
+const AUTH = `${env.REACT_APP_AUTH_API}/api/auth/graphql`;
+const AUTH_FORCE_DEV = false;
+const AUTH_DEV = 'https://bento-dev.bento-tools.org/api/auth/graphql';
```
Comment (AjayDoddapaneni) [inline src/utils/graphqlClient.js:9]: can't we use env variable in here?

File: src/bento/userLoginData.js (hunk 3 above)
Comment (iksheth) [reply src/bento/userLoginData.js:11]: Will make that change.

**Comment by iksheth** [comment]
@AjayDoddapaneni Changes Done ✅

**Review by AjayDoddapaneni — APPROVED** [review]

**Pull request merged**